src/api/PizzaApi.ts

```diff
@@ -24,7 +24,7 @@ export const baseFetch = async <T = any>(path: string, options: { [key: string]:
 };
 
 const reviver: (this: any, key: string, value: any) => any = (key, value) => {
-  if (key === "createdAt" || key === "validatedAt") {
+  if (key === "createdAt" || key === "validatedAt" || key === "cancelledAt") {
```

Comment:
The English word is "canceled" but you can spell the attribute however you want / need.

Reply:
@mojowen maybe you and I need 3 reviewers just for spelling 🤣